services-custom/s3-transfer-manager/pom.xml

```diff
@@ -225,6 +225,25 @@
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>${maven-javadoc-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>generate-javadocs</id>
```

Comment:
Overwrote it here to enforce doctlint. The one defined in the main pom.xml bypasses doclint (`<doclint>none</doclint>`)